src/platform/site-wide/sass/style-consolidated.scss

```diff
@@ -48,4 +51,12 @@ $image-path: "~uswds/src/img";
   @import "merger";
 
   @import "consolidated-patches";
+
+  width: 100%;
+  height: auto;
+  background: $teamsite-dark-blue;
+}
+
+.merger #bg-middle-effect {
+    background: url(/va_files/2014/responsive/images/bg-middle-effect.png) repeat-y scroll center -65px $teamsite-light-blue;
```

Comment:
Only the color has been updated per https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14099#issuecomment-429156261:
![image](https://user-images.githubusercontent.com/16051172/46973879-5cb74380-d077-11e8-88f7-e0b4818e9849.png)

Reply:
In firefox, adding this `$teamsite-light-blue` at the end causes it to only show the blue and never show the image. Not sure if thats expected or not.

Reply:
Thanks @RobAWilkinson, the variable has been replaced with the value.

Reply:
I think the issue that's causing the color to override the background image is the order of the declaration. If the color is set first, followed by the image, it might work as intended.

Reply:
@erikphansen @cehsu it was just a mistake on my part, I thought the image was the entire background, not just the overlay. So when the color changed i thought the image wasnt loading.